---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
bibliography: ../References/references.bib
---

```{=html}
<!--
Overall the intro is on the brief side...which is better than too long, tho maybe could think if somethings lacking...I think maybe add a paragraph about the function of the placenta.  I think you could also add more on the Ilumina array itself since you use this in all your analyses? How it works and the difference between 450K and EPIC.You know alot about it and the data processing....
 - wendy
-->
```
## Dissertation context and overview

The placenta is a key component to the healthy development of the fetus during pregnancy.
At the cellular level, highly specialized placental cells regulate gene expression programs in part through epigenetics.
DNA methylation (DNAm), an epigenetic modification that involves the chemical addition of a methyl group to cytosines, plays an integral role in regulating gene expression at the cellular level.
Most cytosine DNAm occurs at cytosine-guanine sites (CpGs), although non-CpG methylation (i.e. at cytosines followed by thymine, adenine, or another cytosine) can play important roles in development and disease [@jang2017].

Recent developments in the past two decades have allowed the characterization of DNAm at an unprecedented level of resolution in the human genome.
Specifically, advances in microarray technology combined with bisulfite conversion have allowed the measurement of DNAm on the order of hundreds of thousands of CpGs per sample.
Large human population studies of DNAm using microarrays has taken off in popularity, which has in turn spurred significant developments in bioinformatics, allowing researchers to investigate and understand phenotype-relevant signals in this information-rich genomic data.

Understanding the variation in placental DNAm is important to our understanding of the molecular processes that contribute placental biology, which in turn will lead to a better understanding of the role of the placenta in pregnancy.
Although there is great interest in studying DNAm, there are significant technical and biological challenges that can lead to spurious results.
Understanding the contributors to DNAm variability and how to account for them in our analyses will be a key step in moving DNAm research forward.
In this dissertation, I characterized some of the major contributors to placental DNAm, and have developed bioinformatic methods to predict and account for these factors in future placental DNAm research.
These factors that influence placental DNAm, - the subjects of this dissertation include ethnicity and genotype, cell-specificity and cell composition dynamics across gestation, and maternal cell contamination.

## DNA methylation

Despite all cells of the human body being nearly genetically identical, human cell types must coordinate a highly specific gene expression program to fulfill each of their own unique cellular niches.
To achieve cell-specific gene expression programs, epigenetic processes function as the layer of molecular information that interacts with DNA to regulate gene expression.
Types of epigenetic marks include DNAm, histones and their post-translational modifications, non-histone chromatin proteins, higher order chromatin and chromosome organization, and non-coding RNAs.
DNAm is the most widely studied epigenetic mark, owing to its relative stability and availability of cost-efficient technologies.

DNAm at gene promoters is highly anti-correlated with gene expression.
CpG islands, which are genomic regions (\~1000bp long) with high CpG density often found near gene promoters, are often devoid of DNAm [@bird1985].
Methylation of DNA at CpG islands results in the silencing of gene expression, for example through the recruitment of repressive methyl-binding proteins, or by impairing transcription factor binding [@moore2013].
One of the main roles of DNA methylation is to also silence transposable and viral elements, which make up Approximately 45% of the mammalian genome [@schulz2006].
DNAm also serves important roles in regulating genomic imprinting [@elhamamsy2017], which is where a gene's expression occurs from one allele (monoallelic) in a parent-of-origin -specific manner, and in X-inactivation, which is the silencing of one X chromosomes in females [@cotton2009a].

Developmentally, DNAm is highly dynamic and serves as an important mark that is reprogrammed for the generation and establishment of new genomes.
DNA methylation is first erased during the maturation of primordial germ cells, owing to actions of TET enzymes that convert 5' methyl-cytosine (5'mc) to 5' hydroxymethyl-cytosine (5'hmc), and also to passive dilution through successive replicative events coupled with the absence of normal DNAm maintenance activity [@hackett2013; @hargan-calvopina2016].
During gametogenesis, maturation of sperm and egg cells, DNAm and other epigenetic marks are re-established in a highly cell-specific manner.
But, after fertilization, the asymmetric epigenetic marks acquired in the sperm and egg genomes must become equalized, and thus go through another round of reprogramming [@fraser2016].
With the exception of imprinted regions, for which their allele-specificity is protected from this erasure, DNAm is erased in the paternal genome through primarily active processes governed by the TET family of enzymes.
In contrast, the maternal DNAm erasure that occurs through mostly replicative-dependent passive dilution [@xu2021].
Lastly, at the blastocyst stage, DNAm is at it's lowest point, and new epigenetic marks can begin establishing in a highly cell-specific manner, with many cell types and tissues not reaching their final differentiated profiles until later in life.

<!--- Insert Figure 1 --->

(ref:introfig1-cap) DNA methylation reprogramming during development. The first major genome-wide DNA methylation erasure occurs in primordial germ cells (PGCs), and includes erasure of prior gametic imprints. DNA methylation is reestablished, but to a greater extent in sperm as compared to oocytes (egg). After fertilization, the asymmetric paternal and maternal epigenomes begin reprogramming. The paternal genome undergoes active demethylation, whereas the maternal genome undergoes demethylation more gradually through passive replication-dependent mechanisms. Gamete-specific differentially methylated regions (i.e. genomic imprints) and most repetitive sequences are protected during post-fertilization epigenetic reprogramming. At the blastocyst stage, there is the first lineage-specification event of inner cell mass (ICM) and trophectoderm (TE). By the blastocyst stage, DNAm is at its lowest point. After implantation, DNA methylation is established in both ICM and TE, but in TE remains lower, a difference that is retained through development.

```{r introfig1, eval = T, echo = T, echo=FALSE, fig.cap="(ref:introfig1-cap)", fig.scap = "DNA methylation reprogramming during development", fig.align='center',  out.extra=''}
knitr::include_graphics(here::here('images', 'introduction', 'intro-fig1.pdf'))
```

## Placental DNA methylation variation across human populations

Epigenome-wide association studies (EWAS) have shown that a substantial amount of variation in DNAm exists between human populations [@chu2017; @smith2017; @park2018; @adkins2011; @kwabi-addo2010; @liu2010; @fraser2012].
Therefore, if left unaccounted for, population-associated variation can interfere with the discovery of DNAm alterations associated with disease or environment.
This type of confounding, often referred to as population stratification, can be addressed by inferring population-associated variation directly from DNAm data itself [@barfield2014; @rahmani2017; @zhou2017], as is done in genome-wide association studies (GWAS) [@tian2008].
However, unlike genetic markers, epigenetic markers are tissue-specific, and therefore a DNAm-based method developed in a specific tissue or population may not generalize well to other tissues with unique DNAm profiles, such as the placenta.

In EWAS, confounding from population stratification is most often addressed using self-reported ethnicity/race to stratify study samples across the phenotype of interest.
But, defining ethnicity/race is a complex task requiring the interpretation of a combination of biological and social factors leading to several complications: (i) inconsistent definition of ethnicity/race categories between individuals/organizations [@sankar2007; @mersha2015]; (ii) self-reporting more than one ethnicity/race [@kaneshiro2011]; and (iii) missing ethnicity information altogether.
To overcome the limitations of ethnicity/race categories, genetically-defined ancestry can be used [@banda2015] as an alternative measure of population-specific variation.
In contrast to the discrete nature of ethnicity/race categories, genetic ancestry can be expressed as several continuous variables that reflect ancestry composition [@pritchard1999].

### Existing approaches to account for population variation in EWAS

Although the importance of accounting for human population-based variation in EWAS is widely acknowledged, methods for DNAm-specific analysis are underdeveloped.
Barfield 2014 developed a principal components analysis (PCA)-based approach to summarize genetic variation in PC variables that can be later incorporated into downstream analyses [@barfield2014] (Table \@ref(tab:ch3tab1)). 
This approach, which is highly similar and likely inspired from genotype-based methods of assessing genetic ancestry that are routinely used GWAS [@price2006a], was effective in a population of North American Caucasian and African American peripheral blood samples. 
In contrast to genotype-based genetic ancestry, population-specific signal in DNAm data is not nearly as strong; therefore, Barfield 2014 found improvements by using PCA on CpG sites that are nearby (\<50bp) single nucleotide polymorphisms (SNPs), which are where the DNA sequence at a particular genomic position often varies between individuals, and in this case, human populations.
Rahmani 2017 developed an approach they refer to as EPISTRUCTURE, which is also a PCA-based approach using a subset of CpGs. 
In contrast to Barfield 2014's "proximity-based" approach, EPISTRUCTURE uses CpG sites that were directly associated with a nearby SNP. 
These SNPs that are associated with CpG methylation, referred to as methylation quantitative trait loci (mQTLs), were identified in one of the two cohorts included in Rahmani 2017. 
Although EPISTRUCTURE had improvements in capturing population-specific genotype-based variation in their DNAm data, a limitation of this type of approach is that many mQTLs are highly tissue-specific [@delahaye2018; @bell2011; @fraser2012], and moreover SNPs can be highly population-specific [@gibbs2015]. 
Since EPISTRUCTURE was developed in blood and in Europeans, it remains unclear if EPISTRUCTURE can provide reasonable performance in other tissues and populations. 
In contrast to Barfield 2014 and EPISTRUCTURE, Zhou 2017 developed an entirely genotype-based approach by training a random forest predictor of ethnicity, using the 59-65 SNP probes on the 450k and 850k DNAm microarrays [@zhou2017b].
They show that their SNP predictor performs well for at least 3 broad ethnicities in USA, that they defined as White, Black or African American, and Asian.
However, these SNP probes were designed for population-specific analyses in mind, and because of the small number of probes, it is unclear how robust and sensitive to additional populations is Zhou 2017's SNP predictor.

<!--- table on methods to account for population stratification in EWAS -->

(ref:ch2tab1m1) Barfield et al. 2014 [@barfield2014]

(ref:ch2tab1m2) EPI-STRUCTURE [@rahmani2017]

(ref:ch2tab1m3) Zhou et al. 2017 [@zhou2017]

(ref:ch2tab1m4) PlaNET [@yuan2019]

\captionsetup{width=6.5in}

```{r ch2tab1, eval = T, echo = T, echo=FALSE, warning = FALSE, message=FALSE,  out.extra=''}
kableExtra::kbl(
  readr::read_csv(here::here('Tables', 'ch2table1.csv')) %>%
    select(-`Name of method`) %>%
    mutate(method = c('(ref:ch2tab1m1)',
                      '(ref:ch2tab1m2)', 
                      '(ref:ch2tab1m3)',
                      '(ref:ch2tab1m4)')) %>%
    select(method, everything()),
  booktabs = TRUE,
  caption = 'Description of methods to infer self-reported ethnicity or genetic ancestry using HM450K data.') %>%
  kableExtra::kable_styling(full_width = TRUE) %>%
  kableExtra::add_header_above(c(" " = 4, "Sample Characteristics " = 3)) %>%
  kableExtra::footnote(symbol = 'Ethnicity/ancestry as defined in associated study')
```

## Epigenetics and placenta cell types

In the early blastocyst, the majority of gamete-derived epigenetic marks have been erased.
The trophectoderm will develop into one of the layers of the chorion as well as the villous trophoblast, which retains this low methylated state.
Hypomethylation of chorionic villi as a whole reflects that trophoblast is the predominant cell type within this tissue [@grigoriu2011].
In the mouse, trophoblast hypomethylation is associated with down-regulation ofthe DNA methyltransferases, Dnmt3a, Dnmt3b, and Dnmt1a [@oda2013].
However, the role of DNMTs in human trophoblast is less clear [@novakovic2010].
In contrast, the inner cell mass undergoes de novo global DNAm.
These cells are the origin of the fetal tissues, but also contribute to primitive endoderm, from which placental endothelial cells are derived, and to the extraembryonic mesoderm, from which villus stroma is derived.
In addition to differences in DNAm, the early trophectoderm shows higher H3K27 methylation as well as lower levels of histone H2A and/or H4 phosphorylation [@nelissen2011].
Further differentiation processes in both trophoblast and other lineages are associated with additional DNAm changes and histone modifications at specific regulatory loci [@hemberger2007].

The tree-like structures of chorionic villi that compose the placenta consist of an outer layer of stem-like cytotrophoblasts (CTBs), which fuse together to form syncytiotrophoblast (STB).
This syncytium facilitates maternal-fetal exchange and produce hormones to support pregnancy, such as progesterone, leptin, human chorionic gonadotropin, and human placental lactogen.
Transcriptional and epigenetic events control CTB differentiation and are triggered by pregnancy-associated levels in oxygen content [@soncin2015]; @maltepe2005].
Under low oxygen conditions, CTB fusion is impaired, and certain STB-specific CpG sites become hypermethylated [@yuen2013].
For example, syncytin-1, which is critical for STB formation, is down-regulated in response to hypoxia through DNMT3A-dependent hypermethylation [@ruebner2013].
Histone deacetylase complexes (HDAC) and histone acetyltransferases (HAT) are also involved through their effects on the transcription factor GCMa, which regulates syncytin-1 expression [@chuang2006].

The terminally differentiated syncytium contains nuclei derived from the fusion of CTBs at various stages in gestation.
As a result, nuclei in the syncytium are heterogeneous in their age and structure.
Nuclei from recently fused CTBs tend to contain a transcriptionally active euchromatic structure whereas older heterochromatic nuclei form condensed structures called syncytial knots [@burton2009].
The majority of STB nuclei are transcriptionally active and lacking repressive epigenetic marks, such as DNAm, H3K9me3, and H3K27me3 [@fogarty2015].
However, there is also an enrichment of H4K20me3, a repressive chromatin mark, that is suggested to potentially originate from the older nuclei.

CTBs can also differentiate into extravillous trophoblasts (EVTs), which exhibit strikingly different transcriptional, epigenetic, and morphological characteristics compared to STBs.
During this process, expression of cell adhesion molecules and polarity are lost and expression of mesenchymal markers, migratory, and invasive properties are gained [@davies2016].
Similar to cancer cells, EVTs down-regulate E-cadherin by DNA methylation and up-regulate many proteases (e.g., *MMP9*), phosphatidylinositol 30-kinase (*PI3K*), and several autocrine or paracrine loops (e.g., involving *EGF* and *VEGF*) [@ferretti2007] as they transition into a highly invasive phenotype.
Disruption of DNAm by *DNMT* inhibitor 5-aza-dC results in loss of invasive and migratory properties in EVT cells; up-regulation of epithelial markers, such as E-Cadherin, beta-Catenin, and Cadherin18; and down-regulation of mesenchymal genes, such as *VIM* and *CDH2* [@chen2013].
Transcription factors essential to the initiation of this transition process, *SNAIL* and *SLUG*, are also differentially methylated between CTBs and EVTs [@chen2013a].

Histone acetylation is also essential for trophoblast differentiation, and is thought to be controlled by placental-utero oxygen content through the activity of hypoxia-inducible factor 1 (HIF-1) and HDACs [@maltepe2005].
The HDAC inhibitor, trichostatin-A, results in increased histone acetylation and over-expression of tumor suppressor gene *SERPINB5*, leading to reduced EVT motility and invasion [@dokras2002].
In contrast, acetylation of histones H2A and H2B by the CREB-binding protein (CBP) acetyltransferases inhibits the epithelial-mesenchyme transition and promotes epithelial characteristics in mouse studies [@abell2011].
A higher-order chromatin structure mediated by histone modifications in combination with a locus repeat structure was demonstrated to play a role in STB-specific expression within the human growth hormone gene cluster [@tsai2014], illustrating that the DNA sequence itself can affect chromatin structure.

Our knowledge of cell-specific epigenetics is confined primarily to trophoblastic populations.
However, additional placental cell populations are functionally important and may also be implicated in disease.
Chorionic villus cultures yield the predominant growth of fibroblast cells, which havea DNAm profile that is much more similar to the fetal membranes than to whole placental villi [@robinson2015].
Placental-specific macrophages, Hofbauer cells (HBs), lie in the mesenchymal core of the chorionic villi and in comparison to maternal and fetal macrophages, show hypermethylation of many immune response-related and classical macrophage-activation associated genes, possibly as a result of high expression of DNMT1, DNMT3A, and DNMT3B [@kim2012].

Our current understanding of pathological diseases that affect the placenta is contributed predominantly from bulk-tissue profiling studies, which suffer from the limitation that cellular heterogeneity can result in the dilution of cell-specific signals and the creation of spurious ones.
More targeted studies investigating cell-specific signatures will contribute to our understanding of changes associated with normal placental functioning as well as of abnormal ones.

### Cell-specific placental DNA methylation

Because heterogeneous tissues, such as the placenta, are made up of several cell types, each with a distinct DNAm signature, whole-tissue measurements are ultimately an average of the DNAm signatures of the constituent cell types, weighted by their respective frequency in the bulk tissue sample.
Therefore, changes in DNAm measured in complex tissues can often be attributed to variation in cell composition rather than DNAm changes that occur in the constituent cell populations [@holbrook2017].
This makes interpretation of placental DNAm studies difficult until placental DNAm is characterized at a cell-specific resolution.

During the first few cell divisions after fertilization, there is a wave of genome-wide erasure of DNAm, followed by de novo DNAm in the inner cell mass [@smith2012].
Deriving from the inner cell mass are fetal tissues and the mesenchymal core component of the placental chorionic villi.
Within the mesenchymal core, stromal cells (SC) and fetal macrophages called Hofbauer cells (HB) can be seen in the placental stroma as early as 18 days post conception [@benirschke1973], which are thought to derive from mesenchymal stem cells.
HBs are distinct from decidual macrophages and fetal/maternal monocytes [@kim2012]; they display high phenotypic diversity, promoting angiogenesis early in gestation and later participating in the immune response to pathological processes and infection [@reyes2017; @reyes2018].
Placental vasculature is critically important for proper functioning of the placenta, and depends on the development of vessels beneath the trophoblast layer.
These vessels are formed from endothelial cells (EC) that derive from the chorionic mesoderm [@charolidi2019].
Encompassing the mesenchymal core is a thick trophoblast (TB) epithelial cell layer, which displays a hypomethylated profile [@robinson2015].
TBs comprise a set of functionally distinct subtypes, each with their own unique function [@novakovic2017; @yuen2013]: Cytotrophoblasts (CTB) are stem-like cells that harbor regenerative abilities and give rise to the two major subtypes of TB, the extravillous trophoblast (EVT) and the syncytiotrophoblast (STB).
EVT are motile cells that travel to maternal tissue and remodel maternal vasculature, while STB are a multi-nucleated epithelial layer lining the chorionic villi that perform critical roles in hormone production and nutrient transfer.

As a consequence of its distinct developmental origin, dramatic differences in DNAm between placenta and somatic tissues have been observed [@robinson2015].
Globally, the placenta is hypomethylated compared to other tissues, which was originally attributed to reduced methylation of repetitive element DNAm [@cotton2009; @perrin2007], but was later resolved to be primarily due to placental-specific PMDs [@schroeder2013].
PMDs are long regions of intermediate/low DNAm surrounded by regions of higher DNAm that exist in a highly cell-specific fashion [@salhab2018].
It is unclear if these PMDs have a distinct function or are footprints of earlier developmental events between embryonic and extraembryonic tissues.
Parent-of origin specific DNAm, which is associated with genomic imprinting, is also more commonly found in the placenta than other tissues [@renfree2013].
Almost all known imprinted genes are imprinted in the placenta, and many are exclusively imprinted in the placenta [@hanna2016; @court2014; @hamada2016; @sanchez-delgado2016].
Interestingly, a number of placental-specific imprinted genes are polymorphically imprinted [@hanna2016].
It is possible that cellular and genetic heterogeneity can contribute to polymorphic imprinting, as well as variability in DNAm generally.
Supporting this, a significant role for genetic control of placental DNAm variation was recently characterized [@delahaye2018].
These studies have contributed to our understanding of the unique epigenetics of the placenta, but it remains unclear if these features are maintained in all constituent placental cell types or are confined to specific ones.

### Cell deconvolution

Placental DNAm is often studied in the context of health in relation to disease and environmental exposures.
A common study design is the epigenome-wide association study (EWAS) [@flanagan2015], where differentially methylated CpGs (DMCs) are identified in a high-throughput manner, usually with microarray or sequencing based approaches.
However, placental DNAm studies are almost all carried out using whole CV and are therefore subject to challenges of interpretability due to potential cell composition variability [@teschendorff2017].
Unlike other tissues, such as adult blood and umbilical cord blood, addressing cell composition variability in placenta is difficult due to a lack of reference placental DNAm profiles, which enables bioinformatic estimation of cell composition from cellular deconvolution techniques [@teschendorff2017a].
These methods operate by modelling the whole tissue measurements as a weighted sum of cell type -specific DNAm signatures, where the weights correspond to the relative proportion of each constituent cell type in the whole tissue sample, and can be determined using least-squares or non-constrained regression approaches [@teschendorff2017a; @houseman2012; @newman2015].
Without reference DNAm profiles for each cell population, researchers sometimes account for cell composition using reference-free deconvolution methods.
However, the effectiveness of reference-free deconvolution in capturing cell composition variation has not yet been assessed in placenta.

## Maternal signal in placental DNA methylation data

Illumina high-density microarray profiling for DNAm has advanced placental research by identifying epigenetic processes associated with placental function in relation to maternal environment, genetics, and perinatal health [@konwar2019].
Critical to these studies is the development of bioinformatic tools to process and analyze DNAm data.
Recently, sample contamination, which can result in obscured measurements, has been shown to be prevalent in EWAS data [@heiss2018].
Sex mismatches and, mis-annotation of samples is a problem that has been previously demonstrated in other types of genomic studies [@turco2019].
Heiss and Just 2018 used two bioinformatic approaches to determine whether samples measured on the Illumina 450k and EPIC DNAm microarrays (450k) are contaminated or misannotated.
The first approach estimates the amount of mixing of distinct genotypes in a DNAm array sample, by measuring the amount of outlying signal at 59-65 single-nucleotide polymorphism (SNP) probes, which increases as mixing between genetically distinct individuals occur.
The second approach identifies samples that have a pattern of signal intensity on the X and Y chromosome probes that is consistent with mixing of samples of different sex.
This second approach is only sensitive to those cases where the sexes of the mixed samples are distinct Despite most epigenome-wide association studies (EWAS) having some level of data quality control and sample preprocessing.
Heiss and Just 2018 demonstrate that misannotation and sample contamination is relatively common in DNAm studies, but also importantly note that contamination is identifiable by their approaches and therefore addressable in downstream analysis.

In placental research, where embryo-derived chorionic villi are normally sampled, there is an added challenge of avoiding contamination with maternal tissue, as the placenta is directly attached to the maternal uterine wall and bathed in maternal blood [@turco2019].
Because maternal cells from decidual tissue are distinct in both function and epigenetic profile than placental (embryo-derived) cells, contamination from maternal tissues can obscure measurements of DNAm and make interpretation difficult [@price2016].
While small numbers of maternal macrophages are a normal component of placental cell composition throughout gestation [@thomas2020], placental inflammation can sometimes be associated with infiltration of maternal immune cells into the intervillous space and even the villi themselves (e.g. acute chorioamnionitis [@konwar2018]).
Due to these technical and biological factors that can contribute to maternal DNA presenting in placental samples, it can be difficult to discern whether phenotype-associated DNAm profiles are due to epigenetic reprogramming or if simply the underlying cell composition is changing.

Although careful sample processing can generally avoid most maternal contamination, knowing the degree of maternal DNA contribution for each sample would aid downstream analyses and may be relevant to the biological processes of interest.
Despite this, the degree that maternal DNA contamination influences placental studies has been minimally explored.
Wan 2019 describe a set of quality control measures based on clustering DNAm patterns between control placental and maternal tissue samples, and partially methylated domains (PMDs), that were used to infer that early termination pregnancies are especially at risk of contamination during sample processing [@wan2019].
Maternal contamination can also present a risk to cord blood studies.
Morin 2017 assessed maternal contamination in cord blood DNAm data using tissue-specific approaches, such as using highly cord blood -specific sites to predict contaminated and non-contaminated samples [@morin2017].
Interestingly, Morin 2017 explored using quality measurements (i.e. "no-calls") from genotyping data for samples that had also been measured on the Illumina PsychChip (genotyping array) to confirm contamination calls, but in contrast to Heiss 2018, found that the sensitivity of the genotyping data for genotype mixing was poor.

## Research objectives and hypothesis

Previous work in placental DNAm research involves mostly EWAS-type study designs assessing placental phenotypes such as preeclampsia, chorioamnionitis, neural tube defects, and environment.
However, these studies are limited to existing bioinformatic tools, which have not been extensively explored for their appropriateness and effectiveness in placental data.
Moreover, the underlying DNA methylation of the various cell components of the placental chorionic villi has not yet been characterized.

In this dissertation I aimed to characterize and identify the major sources of variation in placental DNAm, and develop bioinformatic approaches to infer, model, and characterize these variables from placental DNAm data directly.
I hypothesize that cell composition, genetic ancestry and ethnicity, and maternal DNA contamination can be identified and predicted from placental chorionic villi DNAm data.
To assess this, I performed:

1.  A study of developing and comparing approaches to predict ethnicity and genetic ancestry from placental DNAm data using multiple placental GEO datasets.

2.  A study of characterizing the DNAm of placental cell types across gestation, and developing a reference to estimate cell composition from placental chorionic villi DNAm.

3.  A study of using bioinformatics to identify maternal signal from placental DNAm data from previously published placental DNAm studies.

These studies contribute to further insights into major contributors of variation in placental DNA methylation, and provide useful bioinformatic approaches for conducting better placental DNA methylation research.
