---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
bibliography: ../References/references.bib
---

<!-- This section is mandatory! -->

Parts of this chapter have been previously published and accepted for publication (see Preface for contribution details):

Robinson WP, Pe√±aherrera MS, Konwar C, Yuan V, Wilson SL. Chapter 13 - Epigenetic Modifications in the Human Placenta.
In: Leung PCK, Qiao J, editors.
Human Reproductive and Prenatal Genetics.
Academic Press; 2019.
p. 293--311.

<!-- Karin michels book chapter-->

## Dissertation context and overview

The placenta is a key component to the healthy development of the fetus during pregnancy.
At the cellular level, highly specialized placental cell populations regulate gene expression programs through epigenetics.
DNA methylation (DNAm), an epigenetic modification involving the chemical addition of a methyl group to cytosines at cytosine-guanine sites (CpGs), plays an integral role in regulating gene expression at the cellular level.

Recent developments in the past two decades have allowed the characterization of DNAm at an unprecedented level of resolution in the human genome.
Specifically, advances in microarray technology combined with bisulfite conversion has allowed the measurement of DNAm on the order of hundreds of thousands of CpGs per sample.
Large human population studies of DNAm using microarrays has taken off in popularity, which has in turn spurred significant developments in bioinformatics, allowing researchers to investigate and understand phenotype-relevant signals in this information-rich genomic data.

Understanding the variation in placental DNAm is important to our understanding of the molecular processes that contribute placental biology, which in turn will lead to a better understanding of the role of the placenta in pregnancy.
Key to the analysis of tissue-specific DNAm is also the corresponding development and improvement of the bioinformatics.

## DNA methylation

## Placental DNA methylation variation across human populations

Epigenome-wide association studies (EWAS) have shown that a substantial amount of variation in DNA methylation (DNAm) exists between human populations [@chu2017; @smith2017; @park2018; @adkins2011; @kwabi-addo2010; @liu2010; @fraser2012].
Therefore, if left unaccounted for, population-associated variation can interfere with the discovery of DNAm alterations associated with disease or environment.
This type of confounding, often referred to as population stratification, can be addressed by inferring population-associated variation directly from DNAm data itself [@barfield2014; @rahmani2017; @zhou2017], as is done in genome-wide association studies (GWAS) [@tian2008].
However, unlike genetic markers, epigenetic markers are tissue-specific, and therefore a DNAm-based method developed in a specific tissue or population may not generalize well to other tissues with unique DNAm profiles.

In EWAS, confounding from population stratification is most often addressed using self-reported ethnicity/race to stratify study samples across the phenotype of interest.
But, defining ethnicity/race is a complex task requiring the interpretation of a combination of biological and social factors leading to several complications: (i) inconsistent definition of ethnicity/race categories between individuals/organizations [@sankar2007; @mersha2015]; (ii) self-reporting more than one ethnicity/race [@kaneshiro2011]; and (iii) missing ethnicity information altogether.
To overcome the limitations of ethnicity/race categories, genetically-defined ancestry can be used [@banda2015] as an alternative measure of population-specific variation.
In contrast to the discrete nature of ethnicity/race categories, genetic ancestry can be expressed as several continuous variables that reflect ancestry composition [@pritchard1999].

Though the use of genetic ancestry could help to better design EWAS, genotyping markers might not be collected in DNAm studies.
In cases where self-reported ethnicity and genetic ancestry information are unavailable, methods have been developed to infer this information directly from DNAm (Table 1) measured on the popular Infinium Human Methylation 450k Beadchip array (HM450K) [@barfield2014; @rahmani2017; @zhou2017; @bibikova2011].
Barfield et al. 2014 [@barfield2014] and EPISTRUCTURE [@rahmani2017] methods both utilize principal components analysis (PCA) on select DNAm sites to infer genetic ancestry.
Since only DNAm sites that are associated with nearby genetic variation are used, these methods produce principal components (PCs) that are often highly correlated with genome-wide genetic variation [@barfield2014; @rahmani2017], and therefore can be used as a measurement of genetic ancestry.
Zhou et al. 2017 [@zhou2017] explored using the set of 65 SNPs measured on HM450K to produce ethnicity/race classifications.
However, it has not been investigated whether these methods perform well in populations and tissues other than the ones they were developed and tested in

## Epigenetics and placenta cell types

In the early blastocyst, the majority of gamete-derived epigenetic marks have been erased [].
The trophectoderm will develop into one of the layers of the chorion as well as the villous trophoblast, which retains this low methylated state.
Hypomethylation of chorionic villi as a whole reflects that trophoblast is the predominant cell type within this tissue [@grigoriu2011].
In the mouse, trophoblast hypomethylation is associated with down-regulation ofthe DNA methyltransferases, Dnmt3a, Dnmt3b, and Dnmt1a [@oda2013].
However, the role of DNMTs in human trophoblast is less clear [@novakovic2010].
In contrast, the inner cell mass undergoes de novo global DNAm.
These cells are the origin of the fetal tissues, but also contribute to primitive endoderm, from which placental endothelial cells are derived, and to the extraembryonic mesoderm, from which villus stroma is derived.
In addition to differences in DNAm, the early trophectoderm shows higher H3K27 methylation as well as lower levels of histone H2A and/or H4 phosphorylation [@nelissen2011].
Further differentiation processes in both trophoblast and other lineages are associated with additional DNAm changes and histone modifications at specific regulatory loci [@hemberger2007].

The tree-like structures of chorionic villi that compose the placenta consist of an outer layer of stem-like cytotrophoblasts (CTBs), which fuse together to form syncytiotrophoblast (STB).
This syncytium facilitates maternal-fetal exchange and produce hormones to support pregnancy, such as progesterone, leptin, human chorionic gonadotropin, and human placental lactogen.
Transcriptional and epigenetic events control CTB differentiation and are triggered by pregnancy-associated levels in oxygen content [@soncin2015]; @maltepe2005].
Under low oxygen conditions, CTB fusion is impaired, and certain STB-specific CpG sites become hypermethylated [@yuen2013].
For example, syncytin-1, which is critical for STB formation, is down-regulated in response to hypoxia through DNMT3A-dependent hypermethylation [@ruebner2013].
Histone deacetylase complexes (HDAC) and histone acetyltransferases (HAT) are also involved through their effects on the transcription factor GCMa, which regulates syncytin-1 expression [@chuang2006].

The terminally differentiated syncytium contains nuclei derived from the fusion of CTBs at various stages in gestation.
As a result, nuclei in the syncytium are heterogeneous in their age and structure.
Nuclei from recently fused CTBs tend to contain a transcriptionally active euchromatic structure whereas older heterochromatic nuclei form condensed structures called syncytial knots [@burton2009].
The majority of STB nuclei are transcriptionally active and lacking repressive epigenetic marks, such as DNAm, H3K9me3, and H3K27me3 [@fogarty2015].
However, there is also an enrichment of H4K20me3, a repressive chromatin mark, that is suggested to potentially originate from the older nuclei.

CTBs can also differentiate into extravillous trophoblasts (EVTs), which exhibit strikingly different transcriptional, epigenetic, and morphological characteristics compared to STBs.
During this process, expression of cell adhesion molecules and polarity are lost and expression of mesenchymal markers, migratory, and invasive properties are gained [@davies2016].
Similar to cancer cells, EVTs down-regulate E-cadherin by DNA methylation and up-regulate many proteases (e.g., *MMP9*), phosphatidylinositol 30-kinase (*PI3K*), and several autocrine or paracrine loops (e.g., involving *EGF* and *VEGF*) [@ferretti2007] as they transition into a highly invasive phenotype.
Disruption of DNAm by *DNMT* inhibitor 5-aza-dC results in loss of invasive and migratory properties in EVT cells; up-regulation of epithelial markers, such as E-Cadherin, beta-Catenin, and Cadherin18; and down-regulation of mesenchymal genes, such as *VIM* and *CDH2* [@chen2013].
Transcription factors essential to the initiation of this transition process, *SNAIL* and *SLUG*, are also differentially methylated between CTBs and EVTs [@chen2013a].

Histone acetylation is also essential for trophoblast differentiation, and is thought to be controlled by placental-utero oxygen content through the activity of hypoxia-inducible factor 1 (HIF-1) and HDACs [@maltepe2005].
The HDAC inhibitor, trichostatin-A, results in increased histone acetylation and over-expression of tumor suppressor gene *SERPINB5*, leading to reduced EVT motility and invasion [@dokras2002].
In contrast, acetylation of histones H2A and H2B by the CREB-binding protein (CBP) acetyltransferases inhibits the epithelial-mesenchyme transition and promotes epithelial characteristics in mouse studies [@abell2011] .
A higher-order chromatin structure mediated by histone modifications in combination with a locus repeat structure was demonstrated to play a role in STB-specific expression within the human growth hormone gene cluster [@tsai2014], illustrating that the DNA sequence itself can affect chromatin structure.

Our knowledge of cell-specific epigenetics isconfined primarily to trophoblastic populations.
However, additional placental cell populations are functionally important and may also be implicated in disease.
Chorionic villus cultures yield the predominant growth of fibroblast cells, which havea DNAm profile that is much more similar to the fetal membranes than to whole placental villi [@robinson2015].
Placental-specific macrophages, Hofbauer cells (HBs), lie in the mesenchymal core of the chorionic villi and in comparison to maternal and fetal macrophages, show hypermethylation of many immune response-related and classical macrophage-activation associated genes, possibly as a result of high expression of DNMT1, DNMT3A, and DNMT3B [@kim2012].

Our current understanding of pathological diseases that affect the placenta is contributed predominantly from bulk-tissue profiling studies, which suffer from the limitation that cellular heterogeneity can result in the dilution of cell-specific signals and the creation of spurious ones.
More targeted studies investigating cell-specific signatures will contribute to our understanding of changes associated with normal placental functioning as well as of abnormal ones.

## DNA methylation in placental cell types

A well- functioning placenta is critical for the healthy development of the fetus during pregnancy.
DNA methylation (DNAm) profiling of the placenta has been increasingly used to characterize underlying processes associated with adverse perinatal outcomes (e.g. maternal preeclampsia, fetal growth restriction and preterm birth) as well as to study the influence of maternal exposures on epigenetic programming.
DNAm is an epigenetic modification that can regulate or respond to changes in gene expression [@pacis2019; @razin1991].
However, because heterogeneous tissues, such as the placenta, are made up of several cell types, each with a distinct DNAm signature, whole-tissue measurements are ultimately an average of the DNAm signatures of the constituent cell types, weighted by their respective frequency in the bulk tissue sample.
Therefore, changes in DNAm measured in complex tissues can often be attributed to variation in cell composition rather than DNAm changes that occur in the constituent cell populations [@holbrook2017].
This makes interpretation of placental DNAm studies difficult until placental DNAm is characterized at a cell-specific resolution.

During the first few cell divisions after fertilization, there is a wave of genome-wide erasure of DNAm, followed by de novo DNAm in the inner cell mass [@smith2012].
Deriving from the inner cell mass are fetal tissues and the mesenchymal core component of the placental chorionic villi (CV).
Within the mesenchymal core, stromal cells (SC) and fetal macrophages called Hofbauer cells (HB) can be seen in the placental stroma as early as 18 days post conception [@benirschke1973], which are thought to derive from mesenchymal stem cells.
HBs are distinct from decidual macrophages and fetal/maternal monocytes [@kim2012]; they display high phenotypic diversity, promoting angiogenesis early in gestation and later participating in the immune response to pathological processes and infection [@reyes2017; @reyes2018].
Placental vasculature is critically important for proper functioning of the placenta, and depends on the development of vessels beneath the trophoblast layer.
These vessels are formed from endothelial cells (EC) that derive from the chorionic mesoderm [@charolidi2019].
Encompassing the mesenchymal core is a thick trophoblast (TB) epithelial cell layer, which displays a hypomethylated profile [@robinson2015].
TBs comprise a set of functionally distinct subtypes, each with their own unique function [@novakovic2017; @yuen2013]: Cytotrophoblasts (CTB) are stem-like cells that harbor regenerative abilities and give rise to the two major subtypes of TB, the extravillous trophoblast (EVT) and the syncytiotrophoblast (STB).
EVT are motile cells that travel to maternal tissue and remodel maternal vasculature, while STB are a multi-nucleated epithelial layer lining the CV that perform critical roles in hormone production and nutrient transfer.

As a consequence of its distinct developmental origin, dramatic differences in DNAm between placenta and somatic tissues have been observed [@robinson2015].
Globally, the placenta is hypomethylated compared to other tissues, which was originally attributed to reduced methylation of repetitive element DNAm [@cotton2009; @perrin2007], but was later resolved to be primarily due to placental-specific partially methylated domains (PMDs) [@schroeder2013].
PMDs are long regions of intermediate/low DNAm surrounded by regions of higher DNAm that exist in a highly cell-specific fashion [@salhab2018].
It is unclear if these PMDs have a distinct function or are footprints of earlier developmental events between embryonic and extraembryonic tissues.
Parent-of origin specific DNAm, which is associated with genomic imprinting, is also more commonly found in the placenta than other tissues [@renfree2013].
Almost all known imprinted genes are imprinted in the placenta, and many are exclusively imprinted in the placenta [@hanna2016; @court2014; @hamada2016; @sanchez-delgado2016].
Interestingly, a number of placental-specific imprinted genes are polymorphically imprinted [@hanna2016].
It is possible that cellular and genetic heterogeneity can contribute to polymorphic imprinting, as well as variability in DNAm generally.
Supporting this, a significant role for genetic control of placental DNAm variation was recently characterized [@delahaye2018].
These studies have contributed to our understanding of the unique epigenetics of the placenta, but it remains unclear if these features are maintained in all constituent placental cell types or are confined to specific ones.

Placental DNAm is often studied in the context of health in relation to disease and environmental exposures.
A common study design is the epigenome-wide association study (EWAS) [@flanagan2015], where differentially methylated CpGs (DMCs) are identified in a high-throughput manner, usually with microarray or sequencing based approaches.
However, placental DNAm studies are almost all carried out using whole CV and are therefore subject to challenges of interpretability due to potential cell composition variability [@teschendorff2017].
Unlike other tissues, such as adult blood and umbilical cord blood, addressing cell composition variability in placenta is difficult due to a lack of reference placental DNAm profiles, which enables bioinformatic estimation of cell composition from cellular deconvolution techniques [@teschendorff2017a].
These methods operate by modelling the whole tissue measurements as a weighted sum of cell type -specific DNAm signatures, where the weights correspond to the relative proportion of each constituent cell type in the whole tissue sample, and can be determined using least-squares or non-constrained regression approaches [@teschendorff2017a; @houseman2012; @newman2015].
Without reference DNAm profiles for each cell population, researchers sometimes account for cell composition using reference-free deconvolution methods.
However, the effectiveness of reference-free deconvolution in capturing cell composition variation has not yet been assessed in placenta.

To address these challenges, in this study we have generated DNAm reference profiles for 4 major human placental cell populations using the 850k Illumina EPIC DNA methylation microarray, which profiles more than 850,000 CpGs.
Our study is the first to characterize the DNAm of major placental cell populations with a high-resolution approach, across first trimester and term placentas.
We show that cell-specific DNAm occurs at thousands of CpG sites, of which a subset can be used to infer cell composition using cellular deconvolution.
Our study underscores the importance of cell-specific approaches in placental studies, especially when measuring epigenetic features such as DNAm.

## Other major sources of variation in placental DNA methylation

## Referencing figures

This text will render with a figure referenced (Figure \@ref(fig:figureTitle)).

```{r figureTitle, eval = T, echo = T, fig.width = 3, fig.height = 2, echo=FALSE, fig.cap="This is the long caption of the figure that will apear in the main text.", fig.scap = "This is the short title for the figure list", out.extra=''}

# Call the image
img <- readPNG("../Images/Professortocat.png")

# Print the image
grid.raster(img)
```
