---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
bibliography: ../References/references.bib
editor_options: 
  chunk_output_type: console
---

## Introduction

In this study, we provide a comprehensive assessment of maternal cell contamination in placental DNAm studies. 
Building on some of the existing bioinformatic approaches described above, we designed a multi-faceted placental-specific approach to assess maternal cell contamination in DNAm data. 

## Methods

DNAm data was obtained from the data repository GEO for 13 placental studies, 3 of which were from our own lab (Table \@ref(tab:ch4tab1)). 
These included studies of acute chorioamnionitis (dataset 1), preeclampsia (datasets 2, 3, 12, 13), neural tube defects (dataset 4), recurrent pregnancy loss (dataset 5), assisted reproductive technologies (dataset 6), early gestation (dataset 7, 10), neurobehavior of newborns (dataset 8), arsenic exposure (dataset 9), and extremely early gestation preterm birth (dataset 11). 
Decidua and maternal blood samples from additional dataset GSE74738 were included for methylome-wide principal components analysis (PCA). 
All studies had collected 1-4 sites of chorionic villi from the fetal facing side of the placenta, which is done to avoid contaminating maternal tissue. 
DNAm data was normalized with BMIQ and noob; and filtered for poor quality, cross-hybridizing, and probes with nearby SNPs according to the 2021 June 15th update of the original Zhou 2017 DNAm annotation [@zhou2017].

```{r echo=FALSE, results='asis'}
    options(knitr.kable.NA = '')
```

### Identifying mixing of genetically distinct DNA samples

To quantify the amount of mixing of genetically distinct DNA in each sample, a mixture model was fit to the 59 SNPs on the 450k/EPIC array using the R package ewastools [@heiss2018].
Mixing of genotypes is indicated by SNP measurements with a high probability for the "outlier" distribution (referred to as the "P(outlier)" value), which corresponds to measurements that deviate from the 3 possible genotypes. 
P(outlier) values range from 0 to 1, and increases with mixing of distinct genotypes (e.g. fetal and maternal DNA).

### Cell composition outliers

We hypothesized that chorionic villi samples significantly contaminated with maternal decidua or blood could be identified from DNAm-based estimates of placental cell composition.
Specifically cell composition inferred from reference-based cell deconvolution [@yuan2021], and PCA clustering with maternal blood and decidua samples, were tested for their association with genotype-based contamination.
This latter approach is a variation of the PCA approach described previously from Wan 2019 [@wan2019], where instead of using all CpG sites, in this study we used a specific subset of highly decidua and blood-specific CpG sites that we identified through linear modelling.
Decidua and blood-specific CpG sites were determined using the R package limma [@ritchie2015], where decidua samples were compared to villi, and then maternal blood was compared to villi; the top 500 hypo- and hyper- methylated CpGs (total was 2000 cpgs for two tissue-specific comparisons) were used for PCA.

### Epigenetic age

Because epigenetic age should be highly increased in decidua and maternal blood (i.e. adult tissues) compared to fetal-derived placental tissue, we hypothesized that estimated epigenetic age may significantly differ between maternally contaminated and non-contaminated placental samples.
To estimate epigenetic age, placental and adult epigenetic age clocks were applied to chorionic villi samples [@horvath2013; @lee2019].

### Sex chromosome analysis

Placental samples from male fetuses that are contaminated with maternal (female) DNA, should appear more "female-like" when examining sex chromosome DNA methylation.
Using the R package ewastools [@heiss2018], DNA methylation intensity measurements on X and Y chromosome probes were normalized to autosomal signal, and then outliers on X and Y chromosome plots were identified in a dataset-specific manner (termed "XY outliers").

\captionsetup{width=6.5in}

```{r ch4tab1, eval = T, echo = T, echo=FALSE, warning = FALSE, message=FALSE,  out.extra=''}
kableExtra::kbl(
  readr::read_csv(here::here('Tables', 'ch4', 'table1.csv')) %>%
    rename(` ` = 2,
           `  ` = 3),
  booktabs = TRUE,
  caption = 'Placental datasets and number of samples with evidence of contamination.') %>%
  
  kableExtra::add_header_above(c(" " = 4, "Contamination measures (n samples)" = 4))%>%
  kableExtra::kable_styling(full_width = TRUE)
```

### 

### Microsatellite genotyping

## Results

There was variability in estimated genotype mixing between studies (Figure \@ref(fig:ch4fig1)A), however a proportion of datasets showed a number of chorionic villi samples with a high P(outlier) value (Table \@ref(tab:ch4tab1)). 
Due to the proximity of maternal tissue during chorionic villi sampling, we hypothesize that much of the observed genotype mixing can be explained by the presence of maternal cells, either from sample processing or from biological processes such as inflammation. 
We noted the number of samples with P(outlier) values greater than 0.15 for each dataset in Table \@ref(tab:ch4tab1). However, other factors can contribute to high P(outlier) values, such as data quality and non-maternal sample contamination (e.g. unintentional mixing of placental samples). 
We found that samples that failed 1 or more Illumina data quality control checks were overall generally low (Figure \@ref(fig:ch4fig1)B), which is unsurprising given that most studies perform at minimum some baseline data processing and quality control. 
However, in 3 out of 13 datasets, samples that failed control checks showed statistically significantly higher P(outlier) values (Figure \@ref(fig:figc1)A), and in two datasets, 5 and 6, the number of samples that failed at least one control check was a large proportion of the dataset -- 6/6 (100%) for dataset 5, and 32/88 (36%) (Table \@ref(tab:ch4tab2)). 
However, we note that Illumina data quality control checks are fairly specific and each of 27 control checks measures a different aspect of DNAm signal quality; as others have also noted, even if samples fail some Illumina controls, other measures of DNAm data quality can appear fine [@heiss2018].
However, samples that failed quality control only accounted for a proportion of samples with high P(outlier) values, leaving the possibility that genotype mixing could be occurring in the remainder of the samples.

<!-- INSERT FIGURE 1 -->

(ref:ch4fig1cap) Estimated DNA contamination in placental DNAm data. Placental DNAm microarray (450k/850k Illumina) from 13 datasets was downloaded. Genotype contamination was estimated using a mixture model approach from ewastools R package. A) P(outlier), a measure of genotype mixing, estimated across all 13 placental datasets. B) Samples that failed 1 or more Illumina quality control checks tended to have higher P(outlier) values.

```{r ch4fig1, eval = T, echo = T, echo=FALSE, fig.cap="(ref:ch4fig1cap)", fig.scap = "Estimated DNA contamination in placental DNAm data", fig.align='center',  out.extra=''}
knitr::include_graphics(here::here('images', 'ch4', 'Figure 1.pdf'))
```

<!-- INSERT TABLE 2 -->

\captionsetup{width=6.5in}

```{r ch4tab2, eval = T, echo = T, echo=FALSE, warning = FALSE, message=FALSE,  out.extra=''}
kableExtra::kbl(
  readr::read_csv(here::here('Tables', 'ch4', 'table2.csv')),
  booktabs = TRUE,
  caption = 'Number of samples that failed 1 or more of 27 Illumina data quality control checks.')
```

To determine what proportion of high P(outlier) placental chorionic villi samples were contaminated with maternal cells, we performed several other analyses based on DNAm variation.

Wan 2019 used a PCA based approach that identified a set of chorionic villi samples that clustered near maternal blood and decidua, indicating that contaminated placental samples show a globally altered methylation profile that was increasingly similar to the contaminating tissues [@wan2019]. 
In this study, we performed a modified version of PCA, where only highly tissue-specific (maternal blood vs villi, decidua vs villi) CpG sites were used. 
We found that PCA using the top 500 hypo- and hyper- methylated CpGs between villi and either maternal blood or decidua (2x tissues = 2000 CpGs total) resulted in the first principal component PC1 (94% variance explained) being able to separate placental and maternal samples (Figure \@ref(fig:ch4fig2)A). 
We found this tissue-specific approach to be more sensitive than using PCA on all CpGs (Figure \@ref(fig:figc2)A). 
We labelled placental chorionic villi samples based on how many standard deviations (SD) away from the chorionic villi mean on PC1, and towards decidua and maternal blood samples (Figure \@ref(fig:ch4fig2)A, Table \@ref(tab:ch4tab1)). 
We found that PC1 was positively associated (p\<0.001, reference category: "x \< mean") with high P(outlier) values (Figure \@ref(fig:ch4fig2)B), indicating that more maternal tissue-like DNAm profiles in placental chorionic villi samples had correspondingly higher genotype mixing. Treating PC1 as a continuous variable, a linear model between P(outlier) versus PC1 was statistically significant (p\<0.001, slope = 3.26, Figure \@ref(fig:ch4fig2)C). 
Interestingly, we observed a concave shape to the nonlinear fit using the "LOESS" regression option in ggplot2). 
This "increasing and then decreasing" relationship of P(outlier) is consistent with a model of genotype mixing, where after a sample becomes more than 50% contamination, the contaminant becomes the primary component of the sample and P(outlier) will start to decrease. More data is needed, however, as two extreme outliers drive the decreasing side of this relationship (Figure \@ref(fig:ch4fig2)C). 
These two outlier samples belonged to dataset 11, and displayed a cell composition profile consistent with high blood contamination (increased nucleated red blood cells and Hofbauer cells, Figure \@ref(fig:ch4fig2)D), which was estimated from DNAm-based placental cell deconvolution [@teschendorff2017; @yuan2021; @houseman2012].
These findings support that tissue-specific clustering based on PCA can be a highly useful approach to identifying tissue-specific contamination.

<!-- INSERT FIGURE 2 -->

(ref:ch4fig2cap) Maternal contamination alters DNAm profiles of placental samples. A) Principal component analysis (PCA) using 2000 tissue-specific CpG sites resulted in PC1 being able to discriminate between placental and maternal tissue samples. B) Categorization of placental samples based on PC1. Samples with higher PC1 values were associated with higher P(outlier) values. C) A positive statistically significant relationship between P(outlier) and PC1 was observed (p\<0.001) using linear regression (green). A non-linear fit is shown in blue. Samples with high PC1 but low P(outlier) are shown in red. D) Cell composition of dataset 11, estimated from DNAm-based placental cell deconvolution. The two outlier samples (red) with high PC1 values are shown having a high immune cell profile. nRBC: nucleated red blood cells, sd: standard deviation, MBD: maternal blood decidua dataset.

```{r ch4fig2, eval = T, echo = T, echo=FALSE, fig.cap="(ref:ch4fig2cap)", fig.scap = "Maternal contamination alters DNAm profiles of placental samples", fig.align='center',  out.extra=''}
knitr::include_graphics(here::here('images', 'ch4', 'Figure 2.pdf'))
```

We found that P(outlier) and tissue-specific PCA were effective in identifying samples containing maternal cell contamination.
We additionally explored other DNAm based variables that might be suggestive of contamination, such as epigenetic age and XY signal intensity analysis.
For epigenetic age, we hypothesized that because of the large difference in epigenetic age between maternal (adult) and placental (fetal) tissue, epigenetic age of maternally contaminated placental samples should display increased epigenetic age estimates.
We applied both adult and placental epigenetic age clocks on all placental chorionic villi samples [@horvath2013; @lee2019].
Adult epigenetic age (EA) clock was not effective in accurately separating GA in placental samples (Figure \@ref(fig:figc3)A).
However, there was a small, but statistically significant association between adult EA and P(outlier) in chorionic villi samples (p \<0.01, slope = 0.0125 P(outlier) / year; Figure \@ref(fig:ch4fig3)A), an effect that was largely driven by two datasets, 1 and 4 (Figure \@ref(fig:figc3)B).
Placental epigenetic gestational age (EGA), which was highly accurate on chorionic villi samples (mean difference = +1.07 weeks, sd = 0.88), yielded estimates of EGA that was not as high as expected for decidua (mean = 43.0 weeks) and maternal blood (mean = 44.2 weeks) samples, which indicated EGA might have low sensitivity for maternal contamination (Figure \@ref(fig:figc3)C).
The difference between EGA and reported was not significantly associated with P(outlier) (Figure \@ref(fig:ch4fig3)B, slope = -0.002, n.s.).

<!-- INSERT FIGURE 3 -->

(ref:ch4fig3cap) Epigenetic age and XY signal intensity analysis. A) Adult epigenetic age on placental chorionic villi samples was associated with P(outlier) (p\<0.01, slope = 0.013). B) The difference between placental epigenetic gestational age and reported gestational age was not significantly associated with P(outlier). C) Total XY signal intensity normalized to total autosomal signal intensity. Male samples (top left) deviations towards female samples (bottom right) are consistent with maternal contamination. D) Males flagged in XY intensity analysis have significantly (p\<0.01) higher P(outlier) values than other placental villi samples.

```{r ch4fig3, eval = T, echo = T, echo=FALSE, fig.cap="(ref:ch4fig3cap)", fig.scap = "Epigenetic age and XY signal intensity analysis", fig.align='center',  out.extra=''}
knitr::include_graphics(here::here('images', 'ch4', 'Figure 3.pdf'))
```

A common approach to check for sample misannotation in DNAm analysis is to compare samples' recorded sex and overall signal intensity from the X and Y chromosome probes.
Additionally, samples that deviate from the male and female sample clusters can be indicative of sample-sample contamination when the contaminant is of different sex.
We hypothesized that maternal contamination could be detected in male pregnancies by XY signal intensity analysis.
We examined XY signal intensity normalized to autosomal signal intensity using ewastools [@heiss2018], and found that 6 datasets had male samples that appear more female-like (trending towards the female cluster) (Figure \@ref(fig:ch4fig3)C).
Although the number of samples flagged in this analysis was small (17/754 males total), flagged samples had significantly increased P(outlier) (p \< 0.01, mean = 0.24 versus 0.04, Figure \@ref(fig:ch4fig3)D), indicating this approach had high specificity but low sensitivity.

Maternal immune cells infiltration into placental tissues can occur in placental inflammatory conditions, such as chorioamnionitis [@cappelletti2020].
Moreover, a (small) proportion of the normal placental composition consists of maternal macrophages [@thomas2020].
In these cases, identified maternal genotype mixing might from variable amounts of maternal cells is a biological variable, not an unwanted technical artifact.
To determine if contamination identified through our approaches were due to biological processes, we tested for the association between our contamination variables and available clinical and placental phenotypes.
We were limited to sample variables made available on GEO, but in almost all cases we were able to test for the main phenotypes for each study (preeclampsia, acute chorioamnionitis, neural tube defects, assisted reproductive technologies).
We found no statistically significant associations between placental phenotype and any of our measures of genetic and maternal contamination.
P(outlier) was only significantly associated with late-onset preeclampsia samples in dataset 2 (p \< 0.05, slope = -0.0084), but not in any of the other 3 preeclampsia datasets.
PC1, generated from our tissue-specific approach described above, was associated with early-onset preeclampsia (p\<0.001, slope = -0.0016), which does not necessarily indicate maternal cell presence given the large methylation alterations associated with severe preeclampsia [@wilson2018].
Overall, we were unable to robustly detect the presence of maternal cells in placental samples due to biological processes, which might indicate our approaches to have inadequate sensitivity to detect the relatively small number of maternal cells that infiltrate due to inflammation.

## Discussion

In this study, we determined if maternal cells could be detected in placental DNAm samples from previous studies.
Although DNAm microarray data was the subject of this study, our findings are relevant to all placental studies where technical contamination from maternal tissue is a challenge, and in studies where maternal cells may be present in placental tissues due to biological processes such as inflammation.
Furthermore, our research supports that contamination in genomic studies in general is a challenge, but can, to an extent, be addressed through bioinformatic data analysis after data has been generated.
Our analyses builds upon the decade of bioinformatics development of DNA methylation -specific array data.
We expect that contamination analyses such as the ones presented in this study, will improve and become more routine as bioinformatics continues to develop, especially with the generation of new genomic technologies (e.g. single-cell).

Our work heavily builds off of two previous works: Heiss 2018, who assessed general sample-sample contamination in DNAm data on GEO [@heiss2018], and Wan 2019, who assessed maternal-specific contamination in placental DNAm studies [@wan2019].
This study improves upon Wan 2019's approach, which used altered DNA methylation patterns associated with maternal contamination, by incorporating additional DNAm-based approaches such as epigenetic age, cell composition, and sex probe intensity analysis; and also by including Heiss 2019's genotype-based contamination approach.
Using this multi-pronged approach, we show that maternal contamination can be reliably identified in a proportion of the 13 placental datasets included in this study.
The dataset -specific variation observed in these placental datasets was expected due to the variability in sample types and sample processing protocols between different research groups, although overall we found the prevalence of highly maternally contaminated placental samples to be infrequent.
Because all data has been previously is from previously published studies, our findings represent a conservative estimate of the prevalence of maternal contamination in placental sampling in general, since highly contaminated samples are likely easily identified and removed through standard QC and data analysis pipelines because of the resulting severely altered DNAm.

We found that the approaches used in this study were variable in their sensitivity, specificity, and therefore their individual usefulness in identifying contamination.
Although high values of P(outlier) (\>0.15) were associated with altered clustering, XY signal intensity, epigenetic (adult) age, we found that some datasets had high P(outlier) samples with no other evidence of maternal contamination.
Because P(outlier) is essentially a measurement of noise and variability of signal intensity, we suspect that dataset-specific variability in background noise will be important to account for when employing this approach, and that dataset-specific considerations can likely important to fine-tune the sensitivity and specificity of these types of analyses.
PCA-based clustering of DNAm samples, like Wan 2019, we found to be useful in identifying chorionic villi samples with more decidua / blood-like DNAm profiles, which, when correlated with genetic contamination, strongly indicates maternal contamination is present.
But in the absence of genotype contamination, deviations on the PC1 variable can be influenced by many other different biological factors, such as cell composition variability, disease, environment, and genetics.
Selection of CpG sites that are highly tissue-specific can amplify the maternal signal and minimize these other sources of variation, as demonstrated by the increased discrimination we showed in our study compared to when using PCA with all array CpGs.
However, this set of CpG sites can be improved since only a low number (n\<13) of maternal decidua and maternal blood samples were used to for CpG selection, making the analysis underpowered.

Other limitations for the other approaches are also important to consider when identifying contamination.
P(outlier) as noted previously, is also function of signal noise, and is not specific to maternal contamination but can be also useful in identifying sample-sample contamination which we did not explore in this study.
Epigenetic age, we found to have little sensitivity to maternal contamination.
We found that the placental age clock [@lee2019] was not able to robustly discriminate maternal decidua and blood samples from villi, to the degree needed to use this as a measure of maternal contamination.
Adult epigenetic age [@horvath2013], was slightly associated with genetic contamination, indicating it has some use in identifying contamination.
XY signal intensity analysis, which Heiss 2018 also explored to assess sample contamination in DNAm data, we found to have a strong association with genetic contamination.
However, the restriction of XY analysis for identifying maternal contamination is only possible in male placental samples, and therefore we limit this approach for use as a secondary validation analysis after applying other contamination checks.
Overall, our study highlights the importance of using a variety of bioinformatic approaches to identify and magnify different aspects maternal contamination from placental DNAm microarray data.

The bioinformatic approaches described in this study to assess different aspects of contamination, although are specific to the Illumina DNAm arrays and to placental tissue, these approaches can be adopted to other genomic data types and to other tissues.
For example, in this study we used CpG sites that were differentially methylated between villi and decidua or maternal blood; however, in cord blood studies, where maternal contamination is also a challenge, a cord-blood specific set of CpG sites could be used instead.
We found that P(outlier) variable based on the 59-65 SNP probes on the 450k/850k methylation arrays to be a fruitful measure of genetic contamination, but this approach could be generalizable to other sets of SNPs such as high-density genotyping arrays, of which many EWAS datasets have available too.
Although, we interestingly Morin 2017 found that genetic contamination measured from genotyping arrays was not correlated with other DNAm-based measures of contamination [@morin2017].
Morin 2017 used the amount of genotype "no-calls", which are when a genotype cannot be confidently discerned from the raw signal intensity array data, as their measure of contamination; this approach is more stringent than the P(outlier) measurements used in this study.
Further studies evaluating SNP data their usefulness in estimating contamination are needed.
