---
output:
  bookdown::pdf_book:
    toc : no
    keep_tex: no
    number_sections: yes
bibliography: ../References/references.bib
---

This chapter is original and unpublished

## Summary and significance of findings

<!-- In this dissertation I investigated... -->

In this dissertation, I investigated some of the major sources of variation of placental DNAm: genetics and human populations, cell type DNAm, and the presence of maternal cells in placental tissues.
Regarding these, I discovered the following insights.
Ethnicity and genetic ancestry can be accurately predicted from placental DNAm microarray data, using both DNAm markers and SNPs.
Many placental-specific DNAm features, such as PMDs, placental-specific imprinting, and repetitive element DNAm, can be found in trophoblast populations, and are often not present in the other constituent placental cell types.
Lastly, I found that maternal contamination can occur in placental samples and can be bioinformatically identified from previous placental DNAm studies.

<!--Chapter 2 -->

The ethnicity predictor trained on 450k placental DNAm data builds on previous studies and adds a placental-specific approach to the toolbox of methods for accounting for population stratification and genetic ancestry in EWAS [@barfield2014; @rahmani2017; @zhou2017]. Although my placental ethnicity predictor, planet, was trained on self-reported ethnicity, I found that the resulting modelled DNAm patterns are highly genotype-dependent, and that my ethnicity predictor is highly correlated with genetic ancestry. In the placental data that I compared planet to other approaches of inferring genetic ancestry and ethnicity, I found that my tissue-specific approach performed better. This affirms this study's original hypothesis, which was that because DNAm is highly tissue-specific, and that genotype-DNAm relationships can also be highly tissue-specific [@delahaye2018], methods to infer biological variables such as ethnicity and genetic ancestry from DNAm are likely also to be tissue-specific.

Self-reported ethnicity has several shortcomings that my DNAm-based ethnicity predictor addresses.
Self-reported ethnicity is often reported for the parents for placental studies, and is often missing from the paternal information.
Self-reported ethnicity is also variably defined between countries, institutions, and individuals.
In research of biological data such as DNAm, self-reported ethnicity is included as a often inadequate [@mersha2015] surrogate of genetic ancestry, which should be accounted for in any population-based study such as the common EWAS design.
Ethnicity, or genetic ancestry, defined based on DNAm or genotyping data, which are reproducible and measurable, addresses these shortcomings of self-reported ethnicity.
It is unclear if DNAm-based genetic ancestry and ethnicity is more appropriate to use in EWAS compared to using genetic ancestry based on genotyping information, but in the absence of genotyping data, I demonstrate that DNAm-based genetic ancestry and ethnicity is a highly useful alternative to self-reported ethnicity.

<!--Chapter 3 -->

Epigenetic modifications such as DNAm have highly cell-specific patterns to regulate cell-specific gene expression.
To investigate the cell-specificity of placental DNAm, I characterized the methylomes of 5 major placental cell types in first trimester and term placentas.
This work builds on previous research that investigated epigenetics in placental cell types, which were previously done using either lower resolution profiling technologies [@grigoriu2011], and / or other epigenetic marks [@fogarty2011; @fogarty2015].
We found highly placental cell-specific DNAm using the 850k Illumina DNAm microarray.
Cell-specific DNAm often occured at promoters and regulatory elements (i.e. enhancers, CpG Islands) near known placental-specific genes, some of which had DNAm that previously had not been characterized before.
Interestingly, we found that for many placental-specific DNAm features, such as PMDs, placental-specific imprinting, and repetitive elements, we found were present in trophoblasts, variably or intermediately present in endothelial and stromal cells, and mostly absent in Hofbauer cells.
Conservation of placental features in trophoblasts is not unsurprising, given that trophoblast make up the bulk of placenta throughout pregnancy.
However, trophoblast themselves is a heterogenous population, with STB becoming the more predominant subtype in later gestation.
Lastly, we assessed cell composition in placental chorionic villi samples using the cell-specific methylomes as a reference for cell deconvolution.
We found that cell composition of chorionic villi between first trimester and term placentas to be highly consistent with previous literature reports from using histology to measure cell composition.

<!--Chapter 4 -->

## Strengths and limitations

Throughout this dissertation, I valued accessibility and open data access practices through several concrete actions.
I implemented user friendly R packages and functions to enable the easy usage of the bioinformatic approaches I developed.
This R package, planet, now includes the placental ethnicity predictor and cell composition reference for deconvolution.
Additionally, I developed the "Placental cell methylome browser", which is a web application that provides a user-friendly interface to access and explore the DNAm data.
Lastly, throughout my dissertation, I made each of my studies' DNAm data available on GEO, for reproducibility and data sharing purposes.
These accessibility practices, I am extremely proud of, and I hope will bring more value to the research I have conducted for my dissertation.

Data integretiy and careful analysis

Open-access data

Collaboration between experts

## Future directions

Incorporating bioinformatic techniques into placental EWAS

## Conclusion

Our understanding of variation in placental DNAm
